api/Dockerfile

```diff
@@ -1,23 +1,24 @@
-FROM abihf/wget:latest AS Downloader
+FROM abihf/wget:latest AS kubeseal-downloader
 RUN wget https://github.com/bitnami-labs/sealed-secrets/releases/download/v0.11.0/kubeseal-linux-amd64 && \
```

Comment:
Yes, we should also consider running kubeseal binary from `$PATH` in the API. This would allow us to easier run tests in local dev environments because we are not bound to the absolute path in the kubseal.py.